Public/Sdk/SelfHost/Libraries/MSBuild/msbuild.dsc

```diff
@@ -36,3 +31,20 @@ export const msbuildRuntimeContent = [
         importFrom("Microsoft.Build.Runtime").Contents.all.getFile(r`contentFiles/any/net472/MSBuild.exe.config`),
     ],
 ];
+
+function getFrameworkFolder() { 
+    return qualifier.targetFramework === "net472" ? a`net472` : a`dotnetcore`; 

```

Comment:
more rubost way if we switch to 48 at some point

return BuildXLSdk.IsNetCore ? "dotnetcore" : qualifier.targetframework

Reply:
Will change

---
In reply to: [301268981](https://github.com/microsoft/BuildXL/pull/561#discussion_r301268981) [](ancestors = 301268981)